graphql_test.go

```diff
@@ -3217,14 +3226,29 @@ func TestInputArguments_failSchemaParsing(t *testing.T) {
 			},
 			Want: want{Error: "field \"Input\": *struct { Thing string } does not define field \"name\" (hint: missing `args struct { ... }` wrapper for field arguments, or missing field on input struct)\n\tused by (*graphql_test.inputArgumentsObjectMismatch3).Hello"},
 		},
+		"Struct field name inclusion": {
+			Args: args{
+				Resolver: &helloOutput{},
+				Opts:     []graphql.SchemaOpt{graphql.UseFieldResolvers()},
+				Schema: `
+					type Query {
+						hello(): HelloOutput!
+					}
+					type HelloOutput {
+						name: Int
```

Comment:
Why are you using an `Int` here when the type of the `Name` field is `string`? I do not understand the problem that you are trying to solve.

Reply:
I tried to explain the problem clearly in the PR message, but let me explain with an example:

Currently, when you have a type mismatch, the error message will say something like:

```
 can not use int64 as Float
        used by (*api4.channelMember).
```

Notice that there is nothing after the `.`? That is because when resolving struct methods, we don't print the field name. We only print `m.Name` always instead of `m.Name` or `sf.Name` depending on the condition. That's what this PR fixes.

Now, with the fix, it will print as:
```
can not use int64 as Float
        used by (*api4.channelMember).LastViewedAt
```

Does that help?